### PR TITLE
Fixed links

### LINKED ISSUE #392: Fix paragraph titles getting covered by top navigation bar when clicked on toc
## Description
When clicking on a table of content link, the title is getting covered by the top navigation bar. 
This happens because the bounding box of the top banner (including the tab) has been changed.

Screenshot 2023-07-06 at 4 16 23 pm

## Why
UI improvements to make users read the titles properly when clicked via toc.

## Close Criteria
- [x] Titles are displayed correctly when clicked from the table of contents.

### PR DESCRIPTION
**Pull request by atteggiani**
Fixes the #392

### CODE REVIEW

**Comment by KAUR1984** [comment]
Thanks for looking into this @atteggiani ! Just noticed that margin from top is only added after I double click the navigation link. Was that something you were intending to fix? Tested on _Safari_ and _Chrome_ browsers. 

https://github.com/ACCESS-Hive/access-hive.github.io/assets/42607679/6f06b201-045e-46f9-a9ae-c1cc96ac9a40

File: docs/js/miscellaneous.js [hunk 1]
```diff
@@ -16,9 +17,35 @@ function removeMkDocs() {
...
+
+// Join all functions
 function main() {
   sortTables();
   removeMkDocs();
+  adjustScrollingToId();
 }
 
+// Run all functions
 window.onload = () => document$.subscribe(() => main());
```
Comment (KAUR1984) [inline docs/js/miscellaneous.js:51]: Whoops, do we need a new line here?

**Comment by KAUR1984** [comment]
> Thanks for looking into this @atteggiani ! Just noticed that margin from top is only added after I double click the navigation link. Was that something you were intending to fix? Tested on _Safari_ and _Chrome_ browsers. Kindly find the attached video below. :)
> 
>  Screen.Recording.2023-07-09.at.06.25.52.mov

Not sure if this is related to how the `click` event works - https://developer.mozilla.org/en-US/docs/Web/API/Element/click_event#usage_notes

**Comment by atteggiani** [comment]
Oh that is actually weird. 
It'a true, but in my deployment with `mkdocs serve` everything works well at first click. 
Cannot understand why there is this difference.

File: docs/js/miscellaneous.js (hunk 1 above)
Comment (atteggiani) [reply docs/js/miscellaneous.js:51]: I don't think so. 
What would you like to add?

**Comment by KAUR1984** [comment]
Oh ok.. I will try and test it today by checking out to your branch locally. :)

**Comment by atteggiani** [comment]
The links don't work properly in the PR because all the links are processed through the RxJS `fromEvent` function, so `event.preventDefault()` seems not to work. 
Not merging until solution found.

**Pull request merged**